packages/datetime/src/dateRangePicker.tsx

```diff
@@ -275,7 +287,7 @@ export class DateRangePicker
         );
     }
 
-    private renderOneMonthCaption() {
+    private renderSingleDayPickerCaption() {
```

Comment:
changed to `singleCaption` to mimic the other `leftCaption` and `rightCaption` methods